plugins/modules/aci_aep.py

```diff
@@ -246,10 +249,19 @@ def main():
             module_object=aep,
             target_filter={'name': aep},
         ),
+        child_classes=['infraProvAcc']
     )
+
     aci.get_existing()
 
+    try:
+        if len(aci.existing[0]['infraAttEntityP']) == 1 and infra_vlan is False:
```

Comment:
I am not sure this condition is correct. I think we should be checking that aci.existing[0]['infraAttEntityP']['children'] does not contain an object 'infraProvAcc'. I have added an ask for a test case below which might trigger my issue with this condition.